### PR TITLE
Moving returners content to Teaching Vacancies

### PR DESCRIPTION
**Pull request by philspray**
### Trello card

https://trello.com/c/juwx8HlK

### Context

A cross-team decision had been made to move the returners' content from GIT to Teaching Vacancies.

### Changes proposed in this pull request

 - redirecting lniks
 - deleting pages
 - removing links and references

### CODE REVIEW

File: app/views/content/blog/getting-ready-to-apply.md [hunk 1]
```diff
@@ -52,7 +52,6 @@ Get one step closer to the classroom with guidance tailored to you, including ho
 An adviser with years of teaching experience can help you if you're (one of the following):
 
 * ready to get into teaching
-* returning to teach at a secondary school in England and already have qualified teacher status (QTS)
```
Comment (EllieNodder) [inline app/views/content/blog/getting-ready-to-apply.md:55]: does this sentence need removing? even though we're removing returners content from GIT, they're still eligible for an adviser, right? @philspray

File: app/views/content/blog/jills-story-getting-back-into-the-classroom.md
Comment (EllieNodder) [inline app/views/content/blog/jills-story-getting-back-into-the-classroom.md:1]: is it a bit of a weird user experience to redirect from a blog to the TV page if they're not going to be migrating the blogs over? would it be better to have some kind of message saying this blog no longer exists but they can find information about returning to teaching at ... ? @philspray

**Review by EllieNodder — COMMENTED** [review]
Do we need to remove this bit from the /teacher-training-advisers page, too?

image

File: app/views/content/blog/getting-ready-to-apply.md [hunk 2]
```diff
@@ -52,7 +52,6 @@ Get one step closer to the classroom with guidance tailored to you, including ho
 An adviser with years of teaching experience can help you if you're (one of the following):
```
Comment (gemmadallmandfe) [inline app/views/content/blog/getting-ready-to-apply.md:52]: This doesn't really make sense now @philspray - there is only one thing in the list!

Need to remove `(one of the following) and merge the bullet into the sentence?

(I know this blog will be deleted soon anyway)

File: app/views/content/non-uk-teachers/_why-teach.html.erb [hunk 3]
```diff
@@ -13,7 +13,6 @@
       </p>
 
       <ul>
-        <li><%= link_to("Read Brooke's story about coming to England to teach", "/blog/my-move-from-canada-to-teach-in-england") %></li>
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/_why-teach.html.erb:16]: Should we keep this for now (and the blog itself and the other international blog), given international content is not moving yet @philspray ?

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ The international relocation payment (IRP) is designed to cover some of the cost
 
 * other relocation expenses
 
-All eligible applicants will receive a single payment of £10,000 from the Department for Education (DfE) around the end of their first term training or working in England. The IRP is a grant – you will not have to pay this money back. You can only receive the IRP once.
+All eligible applicants will receive a single payment of £10,000 from the Department for Education (DfE) around the end of their first term training or working in England. The IRP is a grant – you will not have to pay this money back. You can only receive the IRP once.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:79]: not sure why this file is in here - should it be @philspray ?

File: app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas.md [hunk 5]
```diff
@@ -6,7 +6,6 @@ description: |-
   Return to England after teaching outside the UK and bring your skills and experience back to an English classroom. Get support to make your transition easier.
 related_content:
     Salaries and benefits of teaching in England : "/salaries-and-benefits"
-    Returning to teaching with international experience : "/blog/returning-to-teaching-with-international-experience"
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas.md:9]: GIven that we are not deleting this international returners page just yet, should we keep this link (and the blog itself) @philspray ?

File: app/views/content/teacher-training-advisers/_article.html.erb [hunk 6]
```diff
@@ -92,15 +92,6 @@
       <li>find a role</li>
     </ul>
 
-      <%= render Content::QuoteListComponent.new(
```
Comment (gemmadallmandfe) [inline app/views/content/teacher-training-advisers/_article.html.erb:95]: Shouldn't we remove the section above about returning to teaching as well as the quote @philspray @EllieNodder ?

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 7]
```diff
@@ -504,8 +504,6 @@ keywords:
 
 If you’ve worked as an unqualified teacher, you may be able to get [qualified teacher status (QTS)](/what-is-qts) through an assessment only programme.
 
-If you already have QTS, find out how you can [return to teaching](/returning-to-teaching).
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:507]: Should we keep this link and point to TV @philspray @EllieNodder?

File: config/tags.yml [hunk 8]
```diff
@@ -17,7 +17,6 @@
 - personal statements
 - qualifications
 - references
-- returning to teaching
```
Comment (gemmadallmandfe) [inline config/tags.yml:20]: we should probably add a redirect from `/blog/tag/returning-to-teaching` to the TV page just for completeness @philspray ?

File: spec/features/breadcrumbs_spec.rb [hunk 9]
```diff
@@ -48,8 +48,8 @@
 
     it "includes the current page in plain text" do
       page.within ".breadcrumbs" do
-        is_expected.not_to have_link "Train to be a teacher"
-        is_expected.to have_text "Train to be a teacher"
+        is_expected.not_to have_link "Home Train to be a teacher"
```
Comment (gemmadallmandfe) [inline spec/features/breadcrumbs_spec.rb:51]: @plisovin Just want to check what these changes do and why we need them here? Thanks!

File: spec/features/blog_spec.rb [hunk 10]
```diff
@@ -15,11 +15,13 @@
     expect(page).to have_content("Blog posts about #{tag}")
   end
 
-  shared_examples "paginating blog posts" do |path|
+  shared_examples "paginating blog posts" do |path, count, next_page|
```
Comment (gemmadallmandfe) [inline spec/features/blog_spec.rb:18]: @plisovin Just want to check what these changes do and why we need them here? Thanks!

File: config/images.yml [hunk 11]
```diff
@@ -328,11 +313,6 @@
   variants:
     - "static/content/blog/thumbnails/jessica-walmsley.jpg"
 
-"static/content/blog/brooke-harvey.jpg":
```
Comment (gemmadallmandfe) [inline config/images.yml:331]: Need to keep this if we keep Brooke's blog @philspray ?

File: config/images.yml [hunk 12]
```diff
@@ -360,9 +340,6 @@
 "static/content/blog/thumbnails/stories-generic.jpg":
   alt: Students enjoying a lesson in a classroom
 
-"static/content/blog/thumbnails/stories-katie.jpg":
```
Comment (gemmadallmandfe) [inline config/images.yml:363]: Need to keep this if we keep the international returners blog temporarily @philspray ?

File: app/views/content/blog/getting-ready-to-apply.md (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/content/blog/getting-ready-to-apply.md:55]: I think we remove it - the blog is about getting ready to **apply**

File: app/views/content/blog/jills-story-getting-back-into-the-classroom.md
Comment (gemmadallmandfe) [reply app/views/content/blog/jills-story-getting-back-into-the-classroom.md:1]: I think the pageviews are SO low on returner blogs that we should just keep it simple - there aren't going to be many inbound links to them, and they will drop out of Google search quickly?

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md (hunk 7 above)
Comment (philspray) [reply app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:507]: I was too keen with the delete key!

File: spec/features/breadcrumbs_spec.rb (hunk 9 above)
Comment (plisovin) [reply spec/features/breadcrumbs_spec.rb]: It was test failing previously, and then something has changed again, so I reverted this test in this [commit](https://github.com/DFE-Digital/get-into-teaching-app/pull/3465/commits/592c42275e970474030e2b79ed4fe6c46f48d6ec)

File: app/views/content/blog/returning-to-teaching-with-international-experience.md [hunk 13]
```diff
@@ -66,4 +66,4 @@ My advice would be to:
     Ofsted is the [Office for Standards in Education, Children’s Services and Skills](https://www.gov.uk/government/organisations/ofsted/about) that inspects services providing learning and education.
     The [Ofsted education inspection framework](https://www.gov.uk/government/publications/education-inspection-framework) sets out how they inspect schools in England.
 
-We’re here to answer your questions and provide advice about teaching, whether you’re just thinking about it or you’re ready to apply. [Find out how you can get help and support with getting into teaching](/help-and-support).
+We’re here to answer your questions and provide advice about teaching, whether you’re just thinking about it or you’re ready to apply. [Find out how you can get help and support with getting into teaching](/help-and-support).
```
Comment (gemmadallmandfe) [inline app/views/content/blog/returning-to-teaching-with-international-experience.md:69]: Thanks for reinstating this blog @philspray - just need to remove the `returning to teaching` from it now

image

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged as soon as it has had tech eyes on it

**Review by asatwal — APPROVED** [review]
👍 LGTM - @plisovin I presume with this service there is never a need for an international version of the website in another language, as moving away from translation files means we move away from being able to do that in Rails easily.

**Comment by gemmadallmandfe** [comment]
> 👍 LGTM - @plisovin I presume with this service there is never a need for an international version of the website in another language, as moving away from translation files means we move away from being able to do that in Rails easily.

Thanks very much for looking at this @asatwal - we'll merge it first thing Monday

This content is just for domestic returners - people with QTS who taught in England previously and who are looking to return to teaching - so shouldn't be a need for translation.

In phase two, we'll be looking to move the content for UK citizens looking to return to teaching in England from overseas - again, I'm not sure there is a need for easy translation, but I will raise this with TV as part of that process

**Pull request merged**